pulp_python/app/serializers.py

```diff
@@ -22,6 +23,50 @@ class Meta:
         fields = ('name',)
 
 
+class DistributionDigestSerializer(serializers.ModelSerializer):
+    """
+    A serializer for the Distribution Digest in a Project Specifier
+    """
+    type = serializers.CharField(
```

Comment:
I think not combined is nicer, and we don't need to split when comparing hashes

Reply:
cool. how about a choice field then? Re: https://github.com/pulp/pulp_python/pull/173/files#r192219408, I think we probably do want to only support certain digests. For the moment, we only use digests to filter the content that we download, but it would be good form to check that hash after we download it.

Reply:
This digest is user provided and only used to filter. The hash checking logic would not belong on this but rather on the digest of the created artifact